model/registry/deprecated/container.yaml

```diff
@@ -1,9 +1,10 @@
 groups:
   - id: registry.container.deprecated
+    prefix: container
     type: attribute_group
     brief: "Describes deprecated container attributes."
     attributes:
-      - id: container.labels
+      - id: labels
```

Comment:
should we extract all such changes to the yaml files to dedicated PR?
So these would be not connected to this PR and could be already merged

Reply:
Yes, I was  planning to actually revert these changes as they're uneeded.

Reply:
Reverted.